stl/inc/random

```diff
@@ -790,11 +790,13 @@ protected:
     typename _Swc_Traits::_Cy_t _Carry;
 };
 
+#if !_HAS_CXX17
 template <class _Ty, size_t _Sx, size_t _Rx, class _Swc_Traits>
 const size_t _Swc_base<_Ty, _Sx, _Rx, _Swc_Traits>::short_lag;
 
 template <class _Ty, size_t _Sx, size_t _Rx, class _Swc_Traits>
 const size_t _Swc_base<_Ty, _Sx, _Rx, _Swc_Traits>::long_lag;
+#endif // !_HAS_CXX17

```

Comment:
Now that everything is `static constexpr`, I'd like to explore omitting these separate definitions entirely. (After all, `integral_constant::value` doesn't have such a definition.) Do any problems arise if we drop these definitions?

It appears that GCC requires such definitions in C++14 mode, but MSVC and Clang on Windows don't (presumably this is an MSVC extension). `/permissive-` and `-fno-ms-compatibility` don't appear to affect this.